yolo/export_yolov10.py

```diff
@@ -3,19 +3,122 @@
 
 import re
 import torch
+import torch.nn as nn
 import onnxsim
 import onnx
 from pathlib import Path
 import json
 
-
 from exporter import Exporter
-from ultralytics.nn.tasks import attempt_load_one_weight
+from ultralytics.nn.tasks import temporary_modules, guess_model_task, Ensemble
+from ultralytics.utils.checks import check_suffix, check_requirements
+from ultralytics.utils import DEFAULT_CFG_DICT, DEFAULT_CFG_KEYS, LOGGER, emojis
 from ultralytics.nn.modules import Detect
 from yolo.detect_head import DetectV10
 
+DIR_TMP = "./tmhttps://docs.luxonis.com/en/latest/_static/logo.pngp"
+
+
+def torch_safe_load(weight):
+    """
+    This function attempts to load a PyTorch model with the torch.load() function. If a ModuleNotFoundError is raised,
+    it catches the error, logs a warning message, and attempts to install the missing module via the
+    check_requirements() function. After installation, the function again attempts to load the model using torch.load().
+
+    Args:
+        weight (str): The file path of the PyTorch model.
+
+    Returns:
+        (dict): The loaded PyTorch model.
+    """
+    from ultralytics.utils.downloads import attempt_download_asset
```

Comment:
Should this import be at the beginning of the file or is there a reason it's deferred?

Reply:
No reason, will move it to the beginning of the file.